src/main/java/seedu/address/logic/commands/HelpCommand.java

```diff
@@ -4,20 +4,59 @@
 import seedu.address.commons.events.ui.ShowHelpRequestEvent;
 
 /**
- * Format full help instructions for every command for display.
+ * Format full help instructions for every command for display or command requested
  */
 public class HelpCommand extends Command {
 
+    public static final String COMMAND_ALIAS = "man";
     public static final String COMMAND_WORD = "help";
 
-    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Shows program usage instructions.\n"
-            + "Example: " + COMMAND_WORD;
+    public static final String MESSAGE_USAGE = COMMAND_WORD + "/" + COMMAND_ALIAS + ": "
+            + "Shows program usage instructions.\n"  + "Example: " + COMMAND_WORD;
 
     public static final String SHOWING_HELP_MESSAGE = "Opened help window.";
+    private String commandRequest = null;
+
+
+    //@@author jasmoon
+    public HelpCommand()    {
```

Comment:
[Coding standard](https://oss-generic.github.io/process/codingStandards/CodingStandard-Java.html) violation! One more more of these problems detected:
1. extra/missing spaces between `HelpCommand()` and `{`